pyconbalkan/sponsors/models.py

```diff
@@ -43,3 +44,17 @@ def save(self, force_insert=False, force_update=False, using=None,
             )
         super().save(force_insert=force_insert, force_update=force_update,
                      using=using, update_fields=update_fields)
+
+
+class PackageOption(models.Model):
+    name = models.CharField(max_length=256)
+    description = models.TextField(blank=True, null=True)
```

Comment:
Using null=True for Char / Text fields in Django is generally a bad idea. Is there any specific reason to use it?